libgalois/include/katana/ParallelSTL.h

```diff
@@ -391,6 +393,102 @@ partial_sum(InputIt first, InputIt last, OutputIt d_first) {
   }
 }
 
+template <class InputIt, class OutputIt, class UnaryOperation>
+OutputIt
+transform(
+    InputIt first, InputIt last, OutputIt d_first, UnaryOperation unary_op) {
+  // This will only work for random access iterators
```

Comment:
FWIW, this comment seems a little redundant wrt to the static_assert below

If you want, you could write just:
```cpp
static_assert(...., "transform is only supported on random access iterators");
```

(and throughout)